acceptance/bundle/templates/default-python/fail-missing-uv/script

```diff
@@ -0,0 +1,13 @@
+trace $CLI bundle init default-python --config-file ./input.json --output-dir output
+
+cd output/fail_missing_uv
+trace $CLI bundle validate
+
+# Dynamically find the directory containing 'uv' and remove it from PATH
```

Comment:
can we do something simpler? would set PATH to empty string work?

Reply:
The command is executed via the shell and it still needs to find the shell.